vulture_os/toolkit/api_parser/sentinel_one/sentinel_one.py

```diff
@@ -88,7 +89,10 @@ def _connect(self):
                 ).json()
 
                 assert response.get('data', {}).get('token'), f"Cannot retrieve token from API : {response}"
-                self.session.headers.update({'Authorization': f"Token {response['data']['token']}"})
+                if self.sentinel_one_account_type == "service user" or self.sentinel_one_account_type == "user service":
```

Comment:
1. You added `default = "console"` in forms, is it really compliant with your condition ?
2. You can refactorize with `if self.sentinel_one_account_type in ['service user', 'user service']:`